docs/variants/workstation/installation-manual.md

```diff
@@ -0,0 +1,218 @@
+   The Flash Descriptor Override Strap-Pin is set. Restrictions implied by
+   the Master Section of the flash descriptor are NOT in effect. Please note
+   that Protected Range (PR) restrictions still apply.
+   Enabling hardware sequencing due to multiple flash chips detected.
+   OK.
+   Found Programmer flash chip "Opaque flash chip" (12288 kB,
+      Programmer-specific) mapped at physical address 0x0000000000000000.
+   Reading old flash chip contents... done.
+   Erasing and writing flash chip... Erase/write done.
+   Verifying flash... VERIFIED.
+   ```
+
+   If you get a warning:
+
+   ``` console
+   WARNING! You may be running flashrom on an unsupported laptop.
+   ```
+
+   And programmer initialization failed, run command:
+
+   ``` console
+   flashrom -p internal:laptop=this_is_not_a_laptop -w /tmp/dasharo_workstation_v0.2_rc3.rom --ifd -i bios -i me
+   ```
+
+   If you have placed the jumper correctly you should see the following message
+   in flashrom's output:
+
+   ``` console
+   The Flash Descriptor Override Strap-Pin is set. Restrictions implied by
+   the Master Section of the flash descriptor are NOT in effect. Please note
+   that Protected Range (PR) restrictions still apply.
+   ```
+
+   Newer version of flashrom may not display the warning about unsupported
+   chipset as it already may be marked as tested. Our team has verified that the
+   flashrom updates firmware reliably on this chipset.
+
+## Verification
+
+1. If everything went well (flashrom has verified the flash content),
+2. Shut down machine, move jumper to the original place
+3. Power on the machine.
+4. After rebooting you should see the Dasharo Workstation logo when booting.
+   When logo appears you may press ESC to select the boot device if you want to
+   reboot from other source.
+
+   ![](../../images/dasharo-black.jpg)
+
+## Troubleshooting
+
+If you do not see the logo after few seconds, something probably went wrong or
+a bug was hit. If the LED on the power button shines white, that means the
+platform booted correctly.
+
+![](../../images/white_led.jpg)
+
+If the power button LED shines constantly in orange color, that means you have
+hit an error. The LED will soon to start blinking.
+
+![](../../images/orange_led.jpg)
+
+If you see the logo and after that system do not starts (black screen), please
+take the following steps:
+
+1. Put bootable USB stick to USB port.
+2. Restart computer using power button.
+3. Press ESC key to enter a boot menu.
+4. Choose USB drive from the list.
+5. Re-install operating system.
+
+If you see an flashrom error like this:
+
+``` console
+ERROR: Could not get I/O privileges (Operation not permitted).
+You need to be root.
+Error: Programmer initialization failed.
+```
+
+Use `sudo` with the flashrom command. If the problem persists, probably the
+kernel restricts access to IOMEM. In order to workaround it, append
+`iomem=relaxed` to kernel commandline. You may do it in two ways:
```

Comment:
There's a third, simpler way:
```
    If your computer uses BIOS for booting, then hold down the Shift key while GRUB is loading to get the boot menu.
    If your computer uses UEFI for booting, press Esc several times while GRUB is loading to get the boot menu.
```
and, after getting a GRUB menu, press "e" on a boot entry to append `iomem=relaxed`to kernel commandline and press Ctrl+x or F10 to boot. Although this setting is temporary and will last only during the next boot, this way is faster and a customer doesn't need to re-generate anything.

Reply:
Added